batch.py

```diff
@@ -694,6 +694,20 @@ def crm_pipeline(
     celltype_list = celltypes.split(' ')
     logging.info(f'Cell types to run: {celltype_list}')
 
+    # only run this for relevant genes
+    plink_genes: List[str] = []
```

Comment:
You could make this a set to start, and then you don't need to extend + set at the end. In the end, not a big deal but would be a little cleaner, eg:

```python
_plink_genes = set()
for celltype in celltype_list:
    # ...
    _plink_genes |= set(extract_genes(...))

plink_genes = list(sorted(_plink_genes))
```

Reply:
Also keeps memory usage lower throughout.